scripts/windowsScript.ps1

```diff
@@ -10,10 +10,30 @@ git submodule update --init -- jni/external/faiss
 # As we are using x86_64-w64-mingw32-gcc compiler we need to replace this macro with __MINGW32__
 (Get-Content jni/external/faiss/faiss/impl/index_read.cpp).replace('_MSC_VER', '__MINGW32__') | Set-Content jni/external/faiss/faiss/impl/index_read.cpp
 (Get-Content jni/external/faiss/faiss/impl/index_write.cpp).replace('_MSC_VER', '__MINGW32__') | Set-Content jni/external/faiss/faiss/impl/index_write.cpp
+(Get-Content jni/external/faiss/faiss/impl/platform_macros.h).replace('_MSC_VER', '__MINGW32__') | Set-Content jni/external/faiss/faiss/impl/platform_macros.h
+(Get-Content jni/external/faiss/faiss/impl/platform_macros.h).replace('#define __PRETTY_FUNCTION__ __FUNCSIG__', ' ') | Set-Content jni/external/faiss/faiss/impl/platform_macros.h
```

Comment:
What does this one mean?

Reply:
__FUNCSIG__ is not not present in MINGW32. Hence I need to replace it with blank line.